go-controller/pkg/node/gateway.go

```diff
@@ -268,6 +268,35 @@ func gatewayInitInternal(nodeName, gwIntf, egressGatewayIntf string, gwNextHops
 		return nil, nil, err
 	}
 
+	// Set annotation that determines if options:gateway_mtu shall be set for this node.
+	var disableGatewayMtu bool
+	chkPktLengthSupported, err := util.DetectCheckPktLengthSupport(gatewayBridge.bridgeName)
+	if err != nil {
+		return nil, nil, err
+	}
+	if !chkPktLengthSupported {
+		klog.Warningf("Disabling LRP gateway_mtu as this node does not support check_packet_length action")
```

Comment:
maybe we need a little bit more helpful error message here to a user. Something like:
"OVS does not support check_packet_length action. IP fragmentation or large TCP/UDP payloads may not be forwarded correctly"

Reply:
done